sysarc.tex

```diff
@@ -45,117 +45,55 @@ \section{System Architecture} \label{sec:sysarc}
 
 Overall, the system architecture can be divided into three main namespaces; Observatory, Main Telescope (MT) and Auxiliary Telescope (AT). The Observatory is the highest level and encapsulates both the Main Telescope, Auxiliary Telescope and global components such as the weather station, DIMM, etc. The complete set of components that belong to each of these namespaces can be seen in Figs.~\ref{fig:ocs}, ~\ref{fig:atcscs} and \ref{fig:mtcscs}.
 
+\begin{figure}
+\begin{center}
+\includegraphics[width=0.9\textwidth]{Obs}
+\caption{Hi level observatory architecture with namespaces and observatory-wide CSCs. (preliminary)\label{fig:ocs}}
+\end{center}
+\end{figure}
+
 \begin{figure}
 \begin{center}
 \includegraphics[width=0.9\textwidth]{AT}
-\caption{Complete set of AT CSCs\label{fig:atcscs}}
+\caption{Complete set of AT CSCs (preliminary)\label{fig:atcscs}}
 \end{center}
 \end{figure}
 
 \begin{figure}
 \begin{center}
 \includegraphics[width=0.9\textwidth]{LSST}
-\caption{Complete set of MT CSCs\label{fig:mtcscs}}
+\caption{Complete set of MT CSCs (preliminary)\label{fig:mtcscs}}
 \end{center}
 \end{figure}
 
 \subsection{SalObj - Python and scripting }\label{sect:salobj}
-SalObj is a Python library provides a pythonic and object-oriented interface to create CSCs and Scripts that can be executed by the script queue component (see Sect.~\ref{sect:scriptq}). The library defines two sets of base classes that are mirror to each other, Remote and Controller. A Remote will send commands to and receive telemetry and events from a specific component whereas a Controller will receive commands and publish telemetry and events. In this framework, a CSC is a specialized Controller that is configure to perform some basic actions by default. A high level diagram is provided in \figref{fig:salobj}.
+SalObj is a Python library provides a pythonic and object-oriented interface to create CSCs and SAL Scripts that can be executed by the script queue component (see Sect.~\ref{sect:scriptq}). The library defines two sets of base classes that are mirror to each other, Remote and Controller. A Remote will send commands to and receive telemetry and events from a specific component whereas a Controller will receive commands and publish telemetry and events. In this framework, a CSC is a specialized Controller that is configured to perform some basic actions by default. %A high level diagram is provided in \figref{fig:salobj}.
```

Comment:
I worry that this implies that SalObj has high level support for scripts, but the `BaseScript` class is actually in ts_scriptqueue.

One possibility is to tweak the beginning and perhaps the ending:
...provides a high level interface for SAL components such as CSCs and SAL scripts.
...
SalObj also provides `BaseCsc`, a subclass of `Controller` that handles the standard state transitions and is intended to be used as a parent class for CSC